stubs/defaultConfig.stub.js

```diff
@@ -1,4 +1,5 @@
-module.exports = {
+/** @type {import('tailwindcss/types')} */
+const config = {
```

Comment:
How about adding a `defineConfig` function export: 
```ts
// index.js
module.exports.defineConfig = (config) => config

// types.d.ts
export function defineConfig(config: Config): Config
```

So we can have this `tailwind.config.js`:
```js
// @ts-check
const { defineConfig } = require('tailwindcss')

module.exports = defineConfig({
  content: [],
  presets: [],
  darkMode: 'media', // or 'class'
  plugins: [],
})
```

I think this is a really nice pattern for typed configuration objects, to avoid the JSDOC syntax.

Reply:
We can do that in the future if we need to, but for now I don't see a lot of benefits. We still have to import something and still have to annotate something. Thanks for the suggestion though 👍